### ISSUE TITLE
feat - view or add title/description in the individual short link card

### ISSUE THREAD

**Issue by ankitsridhar16** [dubinc/dub#682]
Hello,

When I create a new short link, the title and description are automatically updated based on the original link (as per the response). However, when we save and view the section where all the generated links are listed by card, it currently doesn't show the title or description. This makes it a bit harder to understand what the link is about. It would be beneficial if we displayed the title or description generated in the individual link card or allowed for the creation of a custom description/title for each link.
<img width="910" alt="Screenshot 2024-03-13 at 9 48 24 AM" src="https://github.com/dubinc/dub/assets/36269838/722a64e2-e04f-4050-a0dc-d0aeb0f04ebc">
<img width="820" alt="Screenshot 2024-03-13 at 9 48 43 AM" src="https://github.com/dubinc/dub/assets/36269838/6ba2c947-32c5-4d7c-b1de-7169d2bc8a34">

**Comment by robkallday** [comment]
Yes this would be helpful. Currently I add the title as a comment and need to hover over the comment icon to see the title. Would be nice to see the title when scrolling through all the links.

**Comment by mohammedyunus2002** [comment]
i would like to work on this issue. Can you specify where do you want tile and description to be rendered

**Comment by steven-tey** [comment]
Good news: We just shipped this today! Thanks a lot for all your feedback 🙏 

https://x.com/steventey/status/1816530501919969706
